### DISCUSSION TITLE
Trouble defining keybinding ctrl+shift+p

### DISCUSSION THREAD

**Discussion by kaddkaka** [ghostty-org/ghostty#3781] [Feature Requests, Ideas]
My config has this single line
```
keybind = ctrl+shift+p=new_split:right
```
But it has no effect when I press it. I just get `2;6u` printed to the prompt. `ctrl+shift+z` prints the same stuff.

The terminal inspector seems to say my keys are sent correctly. (default binding `ctrl+shift+o` works)

Environment: Xubuntu+bash+UK keyboard layout

Switching to SV (Swedish) or US keyboard layout has no effect.

see also: https://discord.com/channels/1005603569187160125/1322638900170063912

**Comment by kaddkaka** [comment]
I also tried to set up bindings for
 ```
ctrl  + equal                          equalize_splits
ctrl  + plus                           equalize_splits
```

But pressing `ctrl` plus the key `=` (or with shift `+`) still has the default behavior of enlarging font size.

**Reply by edrozenberg** [reply]
Tested the following in `~/.config/ghostty/config`:
```
keybind = ctrl+shift+p=new_split:right
keybind = ctrl+equal=equalize_splits
keybind = ctrl+shift+equal=equalize_splits
```
(I don't have a dedicated `+` on my keyboard so using `ctrl+shift+equal` instead)

**Results:**

MacOS Ghostty tip/nightly `Version ddf7173a Build 9187`:
All 3 bindings works as expected

Linux Nixos Gnome GTK Ghostty `1.0.1` release:
First 2 bindings work as expected.
But `keybind = ctrl+shift+equal=equalize_splits` does NOT, prints:
`;5u`

Instead the following works fine on Linux:
`keybind = ctrl+shift+plus=equalize_splits`

Interesting.

**Reply by kaddkaka** [reply]
To me, I had to write the keybinding like `keybind = ctrl+shift+plus=equalize_splits` and invoke it with keys `ctrl` + `shift` + `=`

**Reply by 00-kat** [reply]
That one would be #4007 (albeit it was created after this one).
